lighthub/esp.cpp

```diff
@@ -1,54 +0,0 @@
-#include "options.h"
-#ifdef __ESP__
-#include "esp.h"
-
-ESP8266WiFiMulti wifiMulti;
-WiFiClient ethClient;
-
-char mqtt_password[16];
-
-//default custom static IP
```

Comment:
What reason to remove WIFI Manager from project? In any cases, portal is much more user-friendly way to configure device AP/Password etc then "define"  directive

Reply:
At current level lighthub is targeted to advanced users that compile program themself. So it is much faster and easier to set password once in my_build_flags file, than change it every time you need to burn new firmware to test something. 
Although I could not make wifimanager stable connect to my wifiAP. It fails sometimes and needed to restart esp... 
Now it works much much better without it.

Reply:
done